### PR TITLE
Results page

### PR DESCRIPTION
**Pull request by GeorgesFORTRY**

### CODE REVIEW

**Review by frvanco — DISMISSED** [review]

File: src/components/Fight.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { useLocation } from 'react-router-dom';
+
+export default function Fight() {
+  const location = useLocation();
+
+  return (
+    <div>
+      <img src={location.state.myHero.image.url} alt="héros" />
+      <p>VS</p>
+      <img src={location.state.vsHero.image.url} alt="vshéros" />
```
Comment (LoicBrassart) [inline src/components/Fight.jsx:10]: I don't get where you find these variable, if someone can explain it to me it'd be nice

File: src/components/Results.jsx [hunk 2]
```diff
@@ -1,3 +1,79 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+// import ResultCard from './ResultCard';
```
Comment (LoicBrassart) [inline src/components/Results.jsx]: Deas code

File: src/components/Results.jsx [hunk 3]
```diff
@@ -1,3 +1,79 @@
...
+      stat = 'speed';
+    }
+    if (str > int && str > spe && str > pow) {
+      stat = 'strength';
+    }
+    if (pow > int && pow > str && pow > spe) {
+      stat = 'power';
+    } else {
+      stat = 'intelligence';
+    }
```
Comment (LoicBrassart) [inline src/components/Results.jsx]: Bug spotted: what if `int === spe === str === pow` ?
Fix: give a default attribute to `stat` in the beginning

File: src/components/Results.jsx [hunk 4]
```diff
@@ -1,3 +1,79 @@
...
+          `https://www.superheroapi.com/api.php/10158558049699051/${heroes[i].id}`
+        )
+        .then(({ data }) => {
+          tmpHeroes.push(data);
+        });
+    }
+    setTimeout(() => {
+      setMyHeroes(
+        tmpHeroes.filter(
+          (hero) => parseInt(hero.powerstats[chosenStat], 10) > 90
```
Comment (LoicBrassart) [inline src/components/Results.jsx]: What does this filter do ?

File: src/components/heroes.json [hunk 5]
```diff
@@ -0,0 +1,102 @@
+[
```
Comment (LoicBrassart) [inline src/components/heroes.json]: - I don't get why you need this file
- It's not a component, so it should not be in `src/components`

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some Eviiiiiiils, plz fix !

**Review by frvanco — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**